tests/test_order_Steinhardt.py

```diff
@@ -174,25 +171,39 @@ def test_weighted(self):
             self.assertEqual(len(positions), comp.num_particles)
 
     def test_attribute_access(self):
-        comp = freud.order.Steinhardt(1.5, 6)
+        comp = freud.order.Steinhardt(6)
 
         with self.assertRaises(AttributeError):
             comp.norm
         with self.assertRaises(AttributeError):
             comp.order
 
         (box, positions) = util.make_fcc(4, 4, 4)
-        comp.compute(box, positions)
+        comp.compute(box, positions, query_args={'r_max': 1.5})
 
         comp.norm
         comp.order
 
     def test_soft_cutoff(self):
         (box, positions) = util.make_fcc(4, 4, 4)
 
-        # Use a really small cutoff to ensure that it is used as a soft cutoff
-        comp = freud.order.Steinhardt(0.1, 6, num_neighbors=12, Wl=True)
-        comp.compute(box, positions)
+        comp = freud.order.Steinhardt(6, Wl=True)
+        comp.compute(box, positions, query_args={
+            'mode': 'nearest', 'exclude_ii': True, 'num_neighbors': 12})
+        npt.assert_allclose(np.average(comp.order), PERFECT_FCC_W6, atol=1e-5)
+        npt.assert_allclose(comp.order, comp.order[0], atol=1e-5)
+        self.assertAlmostEqual(comp.norm, PERFECT_FCC_W6, delta=1e-5)
+        comp.compute(box, positions, query_args={
+            'mode': 'nearest', 'num_neighbors': 12})
+        npt.assert_allclose(np.average(comp.order), PERFECT_FCC_W6, atol=1e-5)
+        npt.assert_allclose(comp.order, comp.order[0], atol=1e-5)
+        self.assertAlmostEqual(comp.norm, PERFECT_FCC_W6, delta=1e-5)
+        comp.compute(box, positions, query_args={
+            'exclude_ii': True, 'num_neighbors': 12})
+        npt.assert_allclose(np.average(comp.order), PERFECT_FCC_W6, atol=1e-5)
+        npt.assert_allclose(comp.order, comp.order[0], atol=1e-5)
+        self.assertAlmostEqual(comp.norm, PERFECT_FCC_W6, delta=1e-5)
+        comp.compute(box, positions, query_args={'num_neighbors': 12})
```

Comment:
@vyasr I don't know what to do with these tests. The name of this test isn't accurate anymore. (There isn't an equivalent "soft cutoff" because now it's all determined by the neighbor-finding methods.) However, I would like to keep some kind of validation that the Steinhardt order parameter is giving correct values regardless of how the neighbors are provided. Let me know your thoughts.

Reply:
I agree, let's go ahead and remove these tests. They're not relevant for Steinhardt anymore.